FueledUtils/UIExtensions.swift

```diff
@@ -37,6 +37,26 @@ public extension UILabel {
 		])
 		self.font = UIFont(descriptor: newFontDescriptor, size: self.font.pointSize)
 	}
+
+	public func setText(_ string: String?, animated: Bool) {
```

Comment:
You could just define a new transition context function for observe(context:) instead of this function and very specific `animatedText` and `animatedAttributedText` extensions.

Reply:
@leonty-fueled in that case, that wouldn't work as to work properly the animation requires to use `UIView.transition(with: <view>, ...)` rather than `UIView.animate(...)`. However `setText` is also useful in a non-reactive context, which is why it's available here